src/docs/sphinx/BuildingVariorum.rst

```diff
@@ -7,162 +7,150 @@
  Building Variorum
 ###################
 
-This page provides details on how to build Variorum from source.
+Variorum can be built with ``spack`` or with CMake from source. Building Variorum creates the
+``libvariorum`` library, the ``powmon`` monitoring tool, and variorum examples.
 
-For the shortest path, see :doc:`QuickStart`.
-
-**********
- Overview
-**********
-
-Variorum uses CMake for its build system. Building Variorum creates the
-variorum library.
-
-The CMake variables (ENABLE_MPI={ON,OFF} and ENABLE_OPENMP={ON,OFF}) control
-the building of parallel examples.
-
-The build dependencies for a minimal build with no parallel components require the following:
+********************
+ Build Dependencies
+********************
 
+The build dependencies for a ``minimal build`` with no parallel components require the following:
    -  C
    -  hwloc
    -  jansson
+   -  Access to vendor-specific drivers or modules (see :doc:`HWArchitectures`).
 
-********************
- Build Dependencies
-********************
+For a ``parallel build`` with MPI/OpenMP, Variorum depends on:
+   -  rankstr (only required for MPI/OpenMP builds)
 
-Variorum
-========
+The CMake variables (``ENABLE_MPI={ON,OFF}`` and ``ENABLE_OPENMP={ON,OFF}``) control
+the building of parallel examples. If ``ENABLE_MPI=ON``, an MPI compiler is
+required.
 
-   -  hwloc
-   -  jansson
-   -  rankstr (optional)
 
 hwloc (Required)
-----------------
+================
+
+`hwloc <https://www.open-mpi.org/projects/hwloc/>`_ is an open-source project
+providing a portable abstraction of the hierarchical topology of modern
+architectures.
 
-No dependencies.
+Variorum leverages hwloc for detecting hardware topology. When reading/writing
+a register on a particular hardware thread, hwloc can map that to the correct
+physical socket.
 
 jansson (Required)
-------------------
+==================
 
-No dependencies.
+`jansson <https://digip.org/jansson/>`_ is an open-source C library for
+encoding, decoding and manipulating JSON data.
+
+Variorum leverages JANSSON to provide a JSON-based API that can retrieve power
+data for external tools/software.
 
 rankstr (Optional)
-------------------
+==================
+
+`rankstr <https://github.com/ECP-VeloC/rankstr>`_ is an open-source C library
+providing functions that identify unique strings across an MPI communicator.
 
-No dependencies.
+Variorum leverages rankstr to split a communicator into subcommunicators by
+hostname. The allows for a single control or monitor process in Variorum to
+for example, enforce a power or frequency limit on a node or to print the
+hardware counters once on a node.
 
-*****************
- Getting Started
-*****************
+********************
+ Building with CMake
+********************
 
-Clone the Variorum repo from GitHub:
+Variorum can be built and installed as follows after cloning from GitHub:
 
 .. code:: bash
 
+   sudo apt-get install libhwloc15 libhwloc-dev libjansson4 libjansson-dev
```

Comment:
Should we be including this in the build steps? It seems specific to a given OS.

Reply:
I think @rountree suggested this. I personally think it is a good idea to have it in there because it gets most people up and running quickly, and people can figure out the equivalent yum install packages.

Reply:
I added a comment for this line to specify installing of variorum dependencies.